editions/dev/tiddlers/from Heigele and Jurke/Extending the Store.tid

```diff
@@ -5,6 +5,8 @@ sub.num: 2
 tags: doc
 title: Extending the Store
 
+TODO: delete
+
 <$list filter="[!has[draft.of]has[chapter.of]chapter.of[Extending the Store]tag[doc]sort[sub.num]]">
```

Comment:
It looks like maybe this is part of a scheme for dynamically generating sequences of documentation. If it's not used then I agree it can go.